roles/php-fpm/files/php.ini

```diff
@@ -0,0 +1,1680 @@
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
```

Comment:
How are you getting these ansible autoconfigs? Are they generated by you running that stuff locally or?

Reply:
@BonjeBongo Oops, I forgot about this one. No, they are not automatically generated. I added them in at the top.